nixos/modules/services/misc/languagetool.nix

```
@@ -0,0 +1,82 @@
+{ config, lib, options, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.languagetool;
+in {
+  options.services.languagetool = {
+    enable = mkOption {
+      type = types.bool;
+      example = true;
+      default = false;
+      description = mdDoc ''
+        Start the HTTP LanguageTool server.
+      '';
+    };
+
+    port = mkOption {
+      type = types.port;
+      default = 8081;
+      example = 8081;
+      description = mdDoc ''
+        Port on which LanguageTool listens.
+      '';
+    };
+
+    public = mkOption {
+      type = types.bool;
+      default = false;
+      example = true;
+      description = mdDoc ''
+        If `true` allow the server to be connected from anywhere,
+        otherwise only from the computer it was started from.
+      '';
+    };
```

opinion, feel free to ignore:
```suggestion
    public = mkEnableOption "access from anywhere (rather than just localhost)";
```

---

Should I add  `mdDoc` as above?